CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 2.23.0-rc.0 / 2020-11-20
+
+* [CHANGE] Make the React UI default. #8142
+* [CHANGE] The following metrics were removed/renamed in remote write. #6815
+     - `succeeded_samples_total` was removed and `samples_total` was introduced for all the samples attempted to send.
+     - `sent_bytes_total` was removed and was replaced with `samples_bytes_total` and `metadata_bytes_total`.
+     - `failed_samples_total` -> `samples_failed_total` .
+     - `retried_samples_total` -> `samples_retried_total`.
+     - `dropped_samples_total` -> `samples_dropped_total`.
+     - `pending_samples` -> `samples_pending`.
+* [FEATURE] Allow metric metadata to be propagated via remote write. The following new metrics were introduced: `metadata_total` (total number of metadata sent to remote storage), `metadata_failed_total` (total number of metadata send failed, non recoverable errors), `metadata_retried_total` (total number of metadata send retried after facing recoverable errors), `metadata_bytes_total` (total bytes send for metadata after compression). #6815
+* [ENHANCEMENT] Add a metric `max_samples_per_send` for tracking the `max_samples_per_send` config of remote write. #8102
+* [ENHANCEMENT] Make the snapshot directory name always the same length. #8138
+* [ENHANCEMENT] Reduce allocated memory and latency of Hash method making queries a bit faster with less memory allocation. #8025
+* [ENHANCEMENT] Create a checkpoint only once at the end of all head compactions if there were multiple in a loop avoiding the expensive checkpointing loop. #8067
```

Comment:
This is overly lengthy, if someone wants details they can check the PR. You can end this at compactions.